_posts/randy/2022-07-24-how-video-takes-all-day.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: "How Video Takes All Day"
+author: randy
+permalink: /@randy/how-video-takes-all-day
+image: "assets/img/how-video-takes-all-day.png"
+description: As video content gets shorter in step with modern attention spans, entertainment value needs to go up.  How do you figure out the creative parts and pieces in short order? Fortunately, thousands of media assets are available online, shaving eons off production time.
+tags: tails randy analog-sampling
+---
+
+<video width="720" height="auto" controls style="max-width: 100%">
+   source src="/assets/how-video-takes-all-day-720.mp4" type="video/mp4">
+</video>
+
+**SHORT AND SUITE** 
+
+It used to be that 30 seconds was the smallest video molecule that could survive in the Universe; so dictated by TV advertising executives with broadcast air time slots to sell. This runtime was reduced from the length of [60 second commercials](https://www.forbes.com/sites/toddwasserman/2020/08/10/the-30-second-tv-ad-is-history-now-what){:target="_blank"}originally afforded on broadcast airwaves. Today, 15 and even 6 second [commercial slots](https://www.voices.com/blog/effective_length_for_tv_commercials){:target="_blank"}are common on the television cycle powered clock. 
+
+There is no arbitrary clip length limit on the internet. Streaming services like YouTube can easily neak in short ad snacks; you are forced to watch while fumbling for the ad kill button. 
+
+Video writers and producers of late need to pack whole shows; truncated messages into shorter time frames. Making things smaller is usually harder, but edit sweets will abound to fill demands of the moment - however brief. 
+
+**TIME FILLS SPACE KILLS TIME** 
+
+In video production circles, all runtimes - seconds and minutes are not created equal. One can get lucky capturing a happenstance clip of a [monkey smelling butt-finger](https://www.youtube.com/watch?v=t4A1tWzlJW8{:target="_blank"}), or lose a few years hiding in the jungle trying to film the sequel. Strike the jungle shoot before it starts to sound like a good idea. 
+
+The first consideration in any project should be to **determine an achievable goal within the timeframe assigned.** Analog Sampling's video format outline dictates what goals are going to be practical within our modest project case: 
```

Comment:
It's cool to see a similar goal here that I also hope to push on the software front.

Reply:
lesson to myself